### PR TITLE
Release/1.9.0

### PR DESCRIPTION
**Pull request by KochTobi**

### CODE REVIEW

File: src/main/resources/schemas/nanopore-instrument-output.schema.json [hunk 1]
```diff
@@ -97,6 +97,20 @@
...
     },
+    "fastqgz_file": {
+      "allOf": [
+        {
+          "$ref": "#/definitions/file"
+        },
+        {
+          "properties": {
+            "file_type": {
+              "pattern": ".*fastq.gz$"
```
Comment (jenniferboedker) [inline src/main/resources/schemas/nanopore-instrument-output.schema.json:108]: Didn't we want to only have "fastq.gz"?

Comment (jenniferboedker) [inline src/main/resources/schemas/nanopore-instrument-output.schema.json:108]: Didn't we want to only have "fastq.gz"?

Comment (sven1103) [reply src/main/resources/schemas/nanopore-instrument-output.schema.json:108]: Hm, this should have been reverted. Nice spot!

**Review by jenniferboedker — COMMENTED** [review]
looks good to me

**Review by jenniferboedker — APPROVED** [review]

**Pull request merged**